app/helpers/application_helper/toolbar/container_images_center.rb

```diff
@@ -30,7 +30,7 @@ class ApplicationHelper::Toolbar::ContainerImagesCenter < ApplicationHelper::Too
         button(
           :container_image_tag,
           'pficon pficon-edit fa-lg',
-          N_('Edit Tags for this #{ui_lookup(:table=>"container_images")}'),
+          N_('Edit Tags for the selected items'),
```

Comment:
@moolitayer another option: `'Edit Tags for the selected #{ui_lookup(:tables=>"container_images")}'` (just saying)